src/utils/helpers.ts

```diff
@@ -39,19 +39,46 @@ export const toCamelCase = (str: string) => {
   return pascalCase.charAt(0).toLowerCase() + pascalCase.slice(1);
 };
 
-export const toTitleCase = (str: string) => str.charAt(0).toUpperCase() + str.slice(1);
+/**
+ * Capitalize the first letter of a string
+ *
+ * `toTitleCase('foo') => 'Foo'
+ *
```

Comment:
Minor, the first '`' is unmatched.

We can probably rm this just as easily, I'd expect the first sentence of the JSDoc does it justice
```suggestion
```

Reply:
makes sense! I guess 'this capitalizes things' isn't too ambiguous haha